src/dashboard/app.py

```
@@ -5,80 +5,259 @@
+        st.dataframe(opt_df.style.format({
+            "allocated_budget": "₩{:,.0f}",
+            "expected_gain": "{:.2f}"
+        }), use_container_width=True)
+    else:
+        st.info("예산 최적화 결과가 없습니다.")
+        
+    st.divider()
+    st.subheader("A/B 테스트 결과 요약")
+    if ab_res:
+        overall = ab_res.get("overall_test", {})
+        st.write(f"**캠페인명:** Churn Prevention Overall")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)